docs/contracts.rst

```diff
@@ -519,7 +519,7 @@ Ether (without data). Additionally, in order to receive Ether, the fallback func
 must be marked ``payable``. If no such function exists, the contract cannot receive
 Ether through regular transactions.
 
-In such a context, there is usually very little gas available to the function call (to be precise, 2300 gas),
+In such a context, there is usually very little gas available to the function call (to be precise, 2300 gas). That is in addition to the simple value transfer gas cost of 21000. So in total 22300 gas.
```

Comment:
The 21k gas is not available to the called contract. You just pay 21k gas for the transaction itself. It might have multiple simple value transfers which each provide a  stipend of 2.3k gas.

Reply:
It needs rephrasing, since people get confused and copy around the 2.3k value as being the gas limit for the fallback which makes no sense. What would you propose?

Reply:
You can add a sentence like: Note that the gas required by a transaction (as opposed to an internal call) that invokes the fallback function is much higher, because each transaction charges an additional amount of 21k gas or more for things like signature checking.